content/posts/Jaydip_Dey_Final_Report.md

```diff
@@ -0,0 +1,196 @@
+---
+title: "Jaydip_Dey_Final_Report"
```

Comment:
@jaydip1235 It might be a good idea to include the project name in the title for clarity. You can check the preview here: https://deploy-preview-227--cvblog.netlify.app/posts/jaydip_dey_final_report/

Reply:
included the project name **CircuitVerse Practice Section** as the title